lms/static/coffee/src/instructor_dashboard/membership.coffee

```diff
@@ -265,7 +265,7 @@ class AutoEnrollmentViaCsv
         if student_result.is_general_error
           response_message =  student_result.response
         else
-          response_message = student_result.username + '  ('+ student_result.email + '):  ' + '   (' + student_result.response + ')'
+          response_message = student_result.username + '  (' + student_result.email + '):  ' + '   (' + student_result.response + ')'
```

Comment:
Ah, I see. The double-space was already there; you're adding a space before the `+`.

If you're going to touch this line, let's change the double-space to a single space. Else, let's just not touch this line.